### PR TITLE
fix link hover color

### PR DESCRIPTION
**Pull request by DjamelSoualmi**
Fix links hover color
**Detailed purpose of the PR**

**Result and observation**

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Review by StefanoA1 — APPROVED** [review]

File: packages/@coorpacademy-components/src/atom/link/index.js [hunk 1]
```diff
@@ -64,10 +65,10 @@ class Link extends React.Component {
 
   render() {
     const {skin, history: {createHref = identity} = {}} = this.context;
-    const {skinHover, 'data-name': dataName = 'link', ...aProps} = this.props;
+    const {skinHover, hoverColor, 'data-name': dataName = 'link', ...aProps} = this.props;
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/atom/link/index.js:68]:
```suggestion
    const {skinHover, hoverColor = getOr('#00B0FF', 'common.primary', skin), 'data-name': dataName = 'link', ...aProps} = this.props;
```

File: packages/@coorpacademy-components/src/atom/link/index.js [hunk 2]
```diff
@@ -64,10 +65,10 @@ class Link extends React.Component {
...
     const {href, onClick, className, style: propsStyle, children} = this.props;
     const {hovered} = this.state;
-    const primarySkinColor = getOr('#00B0FF', 'common.primary', skin);
+    const primarySkinColor = hoverColor ? hoverColor : getOr('#00B0FF', 'common.primary', skin);
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/atom/link/index.js:71]:
```suggestion
    const primarySkinColor = hoverColor;
```

**Review by Fausto95 — APPROVED** [review]

File: packages/@coorpacademy-components/src/atom/link/index.js (hunk 2 above)
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/atom/link/index.js:71]:
```suggestion
    const hoverSkinColor = hoverColor || getOr('#00B0FF', 'common.primary', skin);
```

**Review by esa-coorp — CHANGES_REQUESTED** [review]

**Review by Fausto95 — APPROVED** [review]

File: packages/@coorpacademy-components/src/atom/link/index.js [hunk 3]
```diff
@@ -77,7 +83,7 @@ class Link extends React.Component {
     const _hoverStyle =
       skinHover && hovered
         ? {
-            color: primarySkinColor
+            color: hoverSkinColor
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/atom/link/index.js:86]:
```suggestion
            color: hoverColor
```
It could just be this simple

**Review by Fausto95 — APPROVED** [review]

**Pull request merged**